lib/rucio/api/rule.py

```diff
@@ -318,5 +316,4 @@ def move_replication_rule(rule_id, rse_expression, activity, source_replica_expr
 
     return rule.move_rule(rule_id=rule_id,
                           rse_expression=rse_expression,
-                          activity=activity,
-                          source_replica_expression=source_replica_expression)
+                          override=override if override else {})
```

Comment:
Final comment, I promise! I believe there’s no need for this use of the ternary operator.

Reply:
In addition, given that `kwargs` is already built above, why not use that directly?

Reply:
Uhhh, a `**kwargs`, love it! ❤️ I'd actually say this is the right place to overrule the consistency, since there is a gain in readability, and we can change it incrementally in the other places too.

PLEASE be nitpicky, that's what code-reviews are for. I'm not totally happy with the commit message either, if you have some suggestions.